clustered/server/src/main/java/org/ehcache/clustered/server/ClusterTierManagerActiveEntity.java

```diff
@@ -79,8 +79,8 @@
   private final AtomicBoolean reconnectComplete = new AtomicBoolean(true);
```

Comment:
Adding this comment here as I'm not able to comment on the truncated lines.

`storeClientMap` can be removed and `trackedClients` set is redundant. `clientIdMap` has that information already.